.github/workflows/windows.yml

```diff
@@ -96,6 +100,8 @@ jobs:
     runs-on: windows-latest
     steps:
       - uses:  actions/checkout@v1
+        with:
+          fetch-depth: 1
```

Comment:
Oops, almost merged this in.  These lines break `git describe` for Windows release build.